src/Tool-DependencyAnalyser-UI/DADependencyTreePresenter.class.st

```diff
@@ -4,6 +4,7 @@ A PDPackageAnalyzerTreeModel shows all dependent packages from a set of packages
 Class {
 	#name : 'DADependencyTreePresenter',
 	#superclass : 'DAPackageTreePresenter',
+	#classTraits : '{} + TraitedClass',
```

Comment:
I think there is a problem with the Tonel export here. 

I've seen messages about this but I did not follow what causes this and what should be done when this happens.

Reply:
I see... I didn't notice it. Iceberg didn't show it either. I was using a fresh image when working on the PR. Is there something I can do to fix this?